bot/internal/bot/flake.go

```diff
@@ -0,0 +1,56 @@
+package bot
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/gravitational/shared-workflows/bot/internal/github"
+	"github.com/gravitational/trace"
+)
+
+const skipPrefix = "/excludeflake"
+
+// ExcludeFlakes gets the list of test names that can be
+// excluded from flaky test detection for a particular PR.
+// Admin reviewers can exclude tests by commenting on a
+// PR with "/excludeflake Test1 Test2".
+//
+// The result is written to a GitHub Action output parameter
+// named FLAKE_SKIP.
+func (b *Bot) ExcludeFlakes(ctx context.Context) error {
+	skip, err := b.testsToSkip(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	outfile := os.Getenv(github.OutputEnv)
+	err = os.WriteFile(outfile, []byte("FLAKE_SKIP="+strings.Join(skip, " ")), 0644)
```

Comment:
I'm not sure if a space is valid in GHA outputs, but it's the only character that I could come up with that won't occur in a Go test name (even subtest descriptions replace ` ` with `_`).

Reply:
How about tilde `~`? Space is fine too if it works.

Reply:
Good idea, will do.